### PR TITLE
DC + API and Fider URLs

### PR DESCRIPTION
**Pull request by DerekRoberts**

### CODE REVIEW

File: openshift/templates/deploy.dc.yaml [hunk 1]
```diff
@@ -77,12 +81,12 @@ objects:
             - image: ${DOCKER_REGISTRY}/${PROJECT_TOOLS}/${NAME}:${SUFFIX}
               name: ${NAME}
               env:
-                - name: REACT_APP_API_BASE_URL
+                - name: env.api-base-url
```
Comment (Kyubinhan) [inline openshift/templates/deploy.dc.yaml]: We still need to keep the name as REACT_APP_API_BASE_URL, REACT_APP_FIDER_LINK. So no need to change anything here

File: openshift/templates/deploy.dc.yaml [hunk 2]
```diff
@@ -77,12 +81,12 @@ objects:
...
                   valueFrom:
                     configMapKeyRef:
                       name: ${NAME}-${SUFFIX}
                       key: env.api-base-url
-                - name: REACT_APP_FIDER_LINK
+                - name: env.fider-url
```
Comment (Kyubinhan) [inline openshift/templates/deploy.dc.yaml]: Same here

File: .jenkins/Jenkinsfile [hunk 3]
```diff
@@ -64,13 +64,11 @@ pipeline {
...
         }
-        stage('Post-promote (PROD)') {
+        stage('Deploy (PROD)') {
             agent { label 'deploy' }
             steps {
                 script {
-                    sh """
-                        echo "TO DO: echo Tag known good prod images, backup databases, backup settings"
-                    """
+                    sh "PROJ_TARGET=${auzhsi-prod} ./openshift/scripts/oc_deploy.sh PROD apply"
```
Comment (Kyubinhan) [inline .jenkins/Jenkinsfile]: PROD or prod, doesn't matter I guess?

File: openshift/templates/deploy.dc.yaml (hunk 1 above)
Comment (DerekRoberts) [reply openshift/templates/deploy.dc.yaml]: Good catches!  :)

**Pull request merged**